scripts/testAll.js

```diff
@@ -0,0 +1,20 @@
+import { spawnSync } from 'child_process'
+import npmWhich from 'npm-which'
+import resolverDirectories from './resolverDirectories'
+
+let npmPath = npmWhich(__dirname).sync('npm')
+let spawnOptions = {
+    stdio: [process.stdin, process.stdout, process.stderr],
+}
+
+spawnSync(
+    npmPath,
+    ['test'],
+    Object.assign({ cwd: __dirname }, spawnOptions))
```

Comment:
```suggestion
    { cwd: __dirname ...spawnOptions }
```
since we're using babel anyways